visual_studio/a1.vcxproj

```diff
@@ -180,7 +180,7 @@
     </ResourceCompile>
     <Midl>
       <AdditionalIncludeDirectories>$(ProjectDir)\..\ext\glfw\include;$(ProjectDir)\..\ext\gl3w;$(ProjectDir)\..\ext\stb_image;$(ProjectDir)\..\ext\sdl\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <OutputDirectory>$(ProjectDir)/$(IntDir)</OutputDirectory>
+      <OutputDirectory>$(ProjectDir)\$(IntDir)</OutputDirectory>

```

Comment:
These changes going to cause issues with Windows vs Linux?

Reply:
oops, thought I reverted this. this was back when I was fighting to get visual studio to run the game, but I don't think it was related to the actual fix